PySDM/backends/impl_numba/methods/collisions_methods.py

```diff
@@ -59,7 +59,7 @@ def coalesce(  # pylint: disable=too-many-arguments
 
 
 @numba.njit(**{**conf.JIT_FLAGS, **{"parallel": False}})
-def breakup0_compute_mult_transfer(
+def compute_multiplicities_transfer(
```

Comment:
would it sound more natural with `compute_transfer_multiplicities` or `compute_multiplicity_transfer`?

Reply:
I like `compute_transfer_multiplicities`